src/services/ReportService.ts

```
@@ -0,0 +1,476 @@
+  // Get default date ranges for common report periods
+  getDateRangeForPeriod(period: string): { dateFrom: Date; dateTo: Date } {
+    const now = new Date();
+
+    switch (period) {
+      case 'today':
+        return {
+          dateFrom: new Date(now.setHours(0, 0, 0, 0)),
+          dateTo: new Date(now.setHours(23, 59, 59, 999)),
+        };
+      case 'yesterday':
+        const yesterday = subDays(now, 1);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)